did-utils/src/methods/did_peer/method.rs

```diff
@@ -473,25 +474,36 @@ impl DidPeer {
     ///
     /// See https://identity.foundation/peer-did-method-spec/#resolving-a-did
     pub fn expand_did_peer_4(&self, did: &str) -> Result<DIDDocument, DIDPeerMethodError> {
-        // Ensure long format by computing did:peer:4 short form alias
-
+        // shorten the did
```

Comment:
You've removed an important information by changing this comment. The next line actually validates that the argument is a valid long did:peer:4 address, and that should be mentioned.